optuna/study.py

```diff
@@ -880,6 +880,21 @@ def load_study(
     # type: (...) -> Study
     """Load the existing :class:`~optuna.study.Study` that has the specified name.
 
+    Example:
+
+        .. testcode::
+
+            import optuna
+
+            def objective(trial):
+                x = trial.suggest_float("x", 0, 10)
+                return x ** 2
+
+            study = optuna.create_study(storage="sqlite:///example.db", study_name="my_study")
+            study.optimize(objective, n_trials=3)
+
+            loaded_study = optuna.load_study(study_name="my_study", storage="sqlite:///example.db")
+
```

Comment:
Just an idea but it might be pedagogical to demonstrate accessing some property of the loaded study. The example is otherwise quite long in contrast to the actual function call of interest, so this may help emphasizing it. E.g.
```python
assert len(loaded_study.trials) == len(study.trials)
```